features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResourcesSerializerCallback.kt

```diff
@@ -7,5 +7,5 @@
 package com.datadog.android.sessionreplay.internal.recorder.resources
 
 internal interface ResourcesSerializerCallback {
-    fun onReady()
+    fun onReady(resourceId: String?)
```

Comment:
in which case it can be null?

Reply:
If we failed to resolve the resourceId. In that case we still want to decrement the pending images, but not to update the wireframe

Reply:
I would say if the flow is `If we failed to resolve the resourceId` then we maybe should have `onFailed` instead of `onReady` with `null` value, but not a blocker to me.